js/main.js

```diff
@@ -1,13 +1,223 @@
 /* global _ */
-var content = "";
-var error = "";
 var xhr = new XMLHttpRequest();
 
-// construct a quote
+var singularize = function(word) {
+    word = word.trim();
+    if (word.substr(-1) === 's') {
+        return word.slice(0, -1);
+    } else {
+        return word;
+    }
+};
+
+//
+// Things Warren Ellis Calls "My Little X"
+//
+var ellisMyLittleLabel = function(vocabulary) {
+    var label = 'my ';
+    switch (_.random(4)) {
+
+        case 0:
+            // eg, "my sexy mammals of the internet"
+            label += _.sample(vocabulary.sexy) + ' ';
+            label += _.sample(vocabulary.mammals) + ' of ';
+            if (Math.random() < 0.5) {
+                label += _.sample(vocabulary.sexiness) + '.';
+            } else {
+                label += 'the ' + _.sample(vocabulary.internet) + '.';
+            }
+            break;
+
+        case 1:
+            // eg, "my little sex monsters"
+            label += 'little ';
+            label += _.sample(vocabulary.sex) + ' ';
+            label += _.sample(vocabulary.monsters) + '.';
+            break;
+
+        case 2:
+            // eg, "my little mammals of doom"
+            label += 'little ';
+            label += _.sample(vocabulary.mammals) + ' of ';
+            switch (_.random(2)) {
+                case 0:
+                    label += _.sample(vocabulary.sexiness) + '.';
+                    break;
+                case 1:
+                    label += _.sample(vocabulary.ill_omen) + '.';
+                    break;
+                default:
+                    label += 'the ' + _.sample(vocabulary.internet) + '.';
+            }
+            break;
+
+        case 3:
+            // eg, "my little sex mammals of doom"
+            label += 'little ';
+            label += _.sample(vocabulary.sex) + ' ';
+            label += _.sample(vocabulary.mammals) + ' of ';
+            if (Math.random() < 0.5) {
+                label += _.sample(vocabulary.ill_omen) + '.';
+            } else {
+                label += 'the ' + _.sample(vocabulary.internet) + '.';
+            }
+            break;
+
+        default:
+            // eg, "my moist sex mammals"
+            label += _.sample(vocabulary.moist) + ' ';
+            label += _.sample(vocabulary.sex) + ' ';
+            label += _.sample(vocabulary.mammals) + '.';
+    }
+    return label;
+};
```

Comment:
Hahaha, your comments are disgusting and hilarious and so helpful!! Genuine comment here, in both revolted and informed.